.github/workflows/versioning.yml

```diff
@@ -50,7 +50,7 @@ jobs:
               example_name=$(basename "$example")
               echo "Building and generating keys for example: $example_name"
               cd "$example"
-              cargo openvm build --no-transpile
+              # cargo openvm build --no-transpile
```

Comment:
is this intentional? should probably undo

Reply:
Oops, good catch.